klein/app.py

```diff
@@ -320,8 +320,10 @@ def run(self, host, port, logFile=None):
         thing your klein application does.
 
         @param host: The hostname or IP address to bind the listening socket
-            to.  "0.0.0.0" will allow you to listen on all interfaces, and
-            "127.0.0.1" will allow you to listen on just the loopback interface.
+            to.  Examples:
```

Comment:
This should be `Examples::`.  Without the double colon, pydoctor will just mangle this into a paragraph, like “             "0.0.0.0"   -> all interfaces (ipv4)              "::"        -> all interfaces (ipv4+ipv6)              "127.0.0.1" -> just the loopback interface.”